.github/workflows/zeek-matrix.yml

```diff
@@ -7,11 +7,12 @@ on:
 
 jobs:
   pre-commit:
-    runs-on: ubuntu-20.04
-    container: zeekurity/zeek
+    runs-on: ubuntu-latest
```

Comment:
Note: Running this outside of a Zeek container will come back to bite us should we want to introduce an action which depends on a Zeek installation (I could think of e.g., type checking with `mypy` which depends on the exact `zeekpkg` version shipping with Zeek, so we probably do not want to install from pypi).

That's what I prepared for when I originally set things up this way. I am unsure what issues you now ran into, but we probably could deal with them once there is a need.

Reply:
Yes agree — let's revisit this when needed. I ran into a pretty frustrating series of problems with `action/setup-python` in the container. I pulled the plug at glibc version errors when invoking Python. (I've removed my fork so I'm afraid I can't point you at it any more, sigh.)